Lib/unittest/test/testmock/testhelpers.py

```diff
@@ -269,6 +269,14 @@ def test_extended_call(self):
         self.assertEqual(mock.mock_calls, last_call.call_list())
 
 
+    def test_extended_not_equal(self):
+        a = call(x=1).foo
```

Comment:
I know it works due to the implementation, but I'd also validate this works recursively for future changes to the module with 
```python 
a = call(x=1).foo().bar
b = call(x=2).foo().bar
```